custom_components/tuya_local/devices/vacplus_dehumidifier.yaml

```diff
@@ -0,0 +1,145 @@
+name: Vacplus Dehumidifier
+products:
+  - id: yttlroi7nr18vqbo
+    name: Vacplus Dehumidifier
+primary_entity:
+  entity: humidifier
+  class: dehumidifier
+  dps:
+    - id: 1
+      name: switch
+      type: boolean
+      mapping:
+        - dps_val: false
+          icon: "mdi:air-humidifier-off"
+          icon_priority: 1
+        - dps_val: true
+          icon: "mdi:air-humidifier"
+          icon_priority: 4
+    - id: 2
+      name: humidity
+      type: integer
+      range:
+        min: 35
+        max: 80
+      mapping:
+        - step: 5
+    - id: 6
+      name: current_humidity
+      type: integer
+    - id: 5
+      name: mode
+      type: string
+      mapping:
+        - dps_val: dehumidify
+          value: Dehumidify
+        - dps_val: rainy
+          value: Rainy
+        - dps_val: dry
+          value: Dry
+        - dps_val: sleep
+          value: Sleep
+secondary_entities:
+  - entity: switch
+    name: Anion
+    dps:
+      - id: 12
+        type: boolean
+        name: switch
+        optional: true
+  - entity: sensor
+    name: Humidity
+    class: humidity
+    dps:
+      - id: 6
+        type: integer
+        name: sensor
+        unit: "%"
+        class: measurement
+  - entity: sensor
+    name: Temperature
```

Comment:
Remove the name, as the `class: temperature` already selects an appropriate translated name.